api/v1alpha1/nodepool_types.go

```diff
@@ -425,7 +464,7 @@ type PowerVSNodePoolPlatform struct {
 	// +kubebuilder:default=tier1
 	// +kubebuilder:validation:Enum=tier1;tier3
 	// +optional
-	StorageType string `json:"storageType,omitempty"`
+	StorageType PowerVSNodePoolStorageType `json:"storageType,omitempty"`
```

Comment:
why we haven't defined variables like `PowerVSNodePoolTier1StorageType` or `PowerVSNodePoolTier3StorageType`?

Reply:
Since we are not using those properties and using only default value defined, did nt create constants.